### DISCUSSION TITLE
Root page cannot have children

### DISCUSSION THREAD

**Discussion by Vongsi** [statamic/cms#4610] [Troubleshooting] [answered]
Following the document below, seems it advised to set the page's template to blog/index. then there was an error

Any one faced the same error ?

https://statamic.dev/quick-start-guide#blog-index

Blog Index
#
**Next, let's make that blog index page. **_Head back to the control panel and go to the Pages collection_**. Create a new entry and call it "Blog", "My Blog", or even "My CyberBlog" — _just make sure the slug is blog. Set the template to blog/index_.**

![image](https://user-images.githubusercontent.com/1546091/139444219-acb5f5f8-1d3c-4133-985b-e1e275bb852b.png)

**Comment by jelleroorda** [comment, marked as answer]
You can fix this by manually removing any `children` from the first entry in your tree in the `content/trees/collections/pages.yaml` file. If you have your collection set up to have a 'root' element, that element is not allowed to have any children.
